src/qt/walletmodel.h

```diff
@@ -151,7 +151,8 @@ class WalletModel : public QObject
     bool isHDEnabled() const;
     bool upgradeWallet(std::string& upgradeError);
 
-    CAmount getBalance(const CCoinControl* coinControl = nullptr, bool fIncludeDelegated = true) const;
+    CAmount getBalance(const CCoinControl* coinControl = nullptr, bool fIncludeDelegated = true, bool fUnlockedOnly = false) const;
+    CAmount getUnlockedBalance(const CCoinControl* coinControl = nullptr, bool fIncludeDelegated = true) const;
```

Comment:
Minor nit: would be nice to add a comment here. `getUnlockedBalance` return the available unlocked balance only (keyword is 'available').